docs/reference/commandline/dockerd.md

```diff
@@ -1,6 +1,6 @@
 <!--[metadata]>
 +++
-title = "daemon"
+title = "dockerd"
```

Comment:
@wenchma the alias creates a redirect from the old (/engine/reference/commandline/daemon/) to the new location so that people arriving at that location from (e.g.) an old bookmark, or external site get properly redirected